libs/two/directory/src/lib/services/directory/directory.service.ts

```diff
@@ -0,0 +1,224 @@
+                console.warn('File is unaccessible, ignoring...');
+              }
+            });
+          }
+        });
+      }
+    });
+  }
+  /**
+   * Watches an array of files in the source directory
+   *
+   * @memberof DirectoryService
+   */
+  public async watchFilesForChange() {
+    const filePaths: string[] = [];
+    this.files.forEach((value, index) => {
+      filePaths.push(`${this.sourceDir}\\${value}`);
+    });
+
+    fs.access(this.sourceDir, fs.constants.F_OK, (err) => {
+      if (err) {
+        throw err;
+      }
+      try {
+        chokidar
+          .watch(filePaths, {
+            atomic: true,
+            depth: 0,
+            awaitWriteFinish: true
+          })
+          .on('change', (filePath: string, stats: Stats) => {
+            console.log(filePath);
+            const tokens = filePath.split('\\');
+            const fileName = tokens[tokens.length - 1];
+            const workFile = `${this.workDir}\\${fileName}`;
+            fs.copyFile(filePath, workFile, (copyErr) => {
+              if (copyErr) {
+                throw copyErr;
+              }
+              this.notifyValidationService(workFile);
+            });
+          });
+      } catch (error) {
+        throw error;
+      }
+    });
+  }
+
+  /**
+   * Checks the current file extension against a set list of acceptable extensions.
+   *
+   * @param {string} ext
+   * @returns {Promise<boolean>}
+   * @memberof DirectoryService
+   */
+  private async extIsValid(ext: string): Promise<boolean> {
+    const exts = ACCEPTABLE_EXTS;
+    return new Promise((resolve, reject) => {
+      const isValid = exts.includes(ext);
+      return resolve(isValid);
+    });
+  }
+  /**
+   * Determines if the supplied filename matches the RegEx defined in @tamu-gisc/two-shared
+   *
+   * @param {string} filename
+   * @returns {Promise<boolean>}
+   * @memberof DirectoryService
+   */
+  private async filenameIsValid(filename: string): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      return resolve(FILENAME_REGEXP.test(filename));
+    });
+  }
+
+  /**
+   * Informs the validation service that the watched file has been altered
+   *
+   * @param {string} filepath
+   * @memberof DirectoryService
+   */
+  private notifyValidationService(filepath: string): void {
+    // const route = path.extname(filepath).substring(1);
+    const route = `${VALIDATION_SERVICE.protocol}${VALIDATION_SERVICE.host}:${VALIDATION_SERVICE.port}/${VALIDATION_SERVICE.globalPrefix}/${VALIDATION_SERVICE.validation_route}`;
+    const options = {
+      method: 'POST',
+      uri: route,
+      body: {
+        path: filepath
+      },
+      json: true
+    };
+    try {
+      axios
+        .post(route, {
+          path: filepath
+        })
+        .catch((err) => {
+          throw err;
+        });
+      // rp(options).catch((err) => {
+      //   throw err;
+      // });
+    } catch (err) {
+      throw err;
+    }
+  }
+}
```

Comment:
Remove dead code.

Prefer NestJS  `HttpService` which uses Axios under the hood. Removing direct use to the Axios library will clean up a dependency from package.json